components/anestesi.py

```diff
@@ -4,36 +4,36 @@
 from dash.dependencies import Input, Output
 
 
-class AnestesiWidget:
+class Anaesthetic:
```

Comment:
Anesthesia instead of Anaesthetic.

"Anaesthesia means "loss of sensation". Medicines that cause anaesthesia are called anaesthetics."